core/templates/components/copy-url/copy-url.component.html

```diff
@@ -0,0 +1,75 @@
+<div class="uid-copy">
```

Comment:
Please prepend all class names with `oppia-` here and everywhere else.

Reply:
done